src/main/java/com/newrelic/jfr/toevent/JITCompilationMapper.java

```diff
@@ -26,8 +27,14 @@
     var timestamp = event.getStartTime().toEpochMilli();
     var duration = event.getDuration();
     var attr = new Attributes();
+    var method = (RecordedMethod) event.getValue("method");
+    if (method != null) {
+      var methodDesc = method.getType().getName() + "." + method.getName() + method.getDescriptor();
+      attr.put("desc", methodDesc);
+    } else {
+      attr.put("desc", "[missing]");
+    }
```

Comment:
I see this in this PR a few times now.  Let's factor out a utility method somewhere so it can be used in all these places, and will read something like `attr.put("desc", describeMethod(method))`.

Reply:
Fixed.